config/i18n/all-langs.js

```diff
@@ -32,7 +34,8 @@ const langDisplayNames = {
 const langCodes = {
   english: 'en-US',
   espanol: 'es-419',
-  chinese: 'zh'
+  chinese: 'zh',
```

Comment:
I am wondering if this should actually be:

```suggestion
  chinese: 'zh'-CN,
```

Reply:
According to https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry (which comes from the MDN docs), `zh-CN` isn't listed as a valid option. Simplified Chinese would be `zh-Hans` it seems.